Source/WebCore/html/HTMLElement.cpp

```diff
@@ -1412,7 +1412,10 @@ ExceptionOr<void> HTMLElement::togglePopover(std::optional<bool> force)
     if (popoverData() && popoverData()->visibilityState() == PopoverVisibilityState::Showing && !force.value_or(false))
         return hidePopover();
 
-    return showPopover();
+    if (force.value_or(true))
+        showPopover();
```

Comment:
We should be doing `return showPopover();`, so it throws all the needed exceptions (including the ones on disconnected elements), that should fix the remaining failures.